src/components/CourseOutlineStatus/index.jsx

```diff
@@ -45,17 +47,22 @@ export default class CourseOutlineStatus extends React.Component {
   }
 
   componentWillReceiveProps(nextProps) {
+    let courseData;
     if (Object.keys(nextProps.courseLaunchData).length > 0) {
-      const checks = getFilteredChecklist(
-        launchChecklist.data, nextProps.courseLaunchData.is_self_paced);
+      courseData = {
+        is_self_paced: nextProps.courseLaunchData.is_self_paced,
+        certificates: nextProps.courseLaunchData.certificates,
+      };
+      const filteredCourseLaunchChecks = getFilteredChecklist(launchChecklist.data,
+        { data: courseData });
```

Comment:
Actually - I think I understand not just sending over `this.props` as the lone variable because of calls like this where we're using data that isn't coming from `nextProps` for the first parameter. I'm still weary of calling the second parameter `props` if we're not sending over the React props value as a whole, but I think that I do understand having it be two different parameters.

Reply:
I think we can rename the variable `props`.